### PR TITLE
Wrench Roll Major update

### PR DESCRIPTION
**Pull request by cndhart**
Add pics, and text—colors pricing s&h

### CODE REVIEW

File: content/toolrolls.md [hunk 1]
```diff
@@ -7,10 +7,37 @@ images:
  - "/img/Std_Met_Shor.jpg"
  - "/img/stdflat.jpg"
  - "/img/tool3.jpg"
- - "/img/SHORTY.jpg"
+ - "/img/STANDARD.jpg"
  - "/img/METRIC.jpg"
+ - "/img/SHORTY.jpg"
+ ordering: true
+orderform will be added soon.  Until then send order via email to cindy@cindycolorado.com please state second color choice when ordering
```
Comment (xaque208) [inline content/toolrolls.md]: This line needs to go outside the `---`, so just below this.

**Comment by cndhart** [comment]
ok did that…I think

mom
Cindy
cindy@cindycolorado.com

File: content/toolrolls.md [hunk 2]
```diff
@@ -11,8 +11,9 @@ images:
  - "/img/METRIC.jpg"
  - "/img/SHORTY.jpg"
  ordering: true
-orderform will be added soon.  Until then send order via email to cindy@cindycolorado.com please state second color choice when ordering
 ---
+orderform will be added soon.  Until then send order via email to cindy@cindycolorado.com please state second color choice when ordering
```
Comment (xaque208) [inline content/toolrolls.md]: This should probably be capitalized too.

**Comment by cndhart** [comment]
The whole thing? or just the UNTIL THEN……………….

Cindy
cindy@cindycolorado.com

File: content/toolrolls.md (hunk 2 above)
Comment (xaque208) [reply content/toolrolls.md]: I just mean the O, in Orderform.

**Comment by cndhart** [comment]
Ok, got it, I’ll fix that right now

mom
Cindy
cindy@cindycolorado.com

**Comment by cndhart** [comment]
Ok I did that.  Somehow I’m not sure about what I am doing with the branch and comment part of this.  Am I doing what I am supposed to do?

Mom
Cindy
cindy@cindycolorado.com

**Comment by xaque208** [comment]
This all looks good.  I'm at the website and commenting there.  You can see it if you visit https://github.com/cindycolorado/cindycolorado.com/pull/7

**Comment by cndhart** [comment]
Ok, I clicked PUBLISH on GitHub…was that something that I should not have done?

Cindy
cindy@cindycolorado.com

**Comment by xaque208** [comment]
Nope, that's all fine.  The site should have deployed now.

**Comment by cndhart** [comment]
Hasn’t yet, I just checked.  This is fun, but somehow I feel like I’m flying on the seat of my pants…..or yours.  Thanks for helping me learn.

Love you,

Mom
Cindy
cindy@cindycolorado.com

**Pull request merged**